opendbc/car/tesla/carstate.py

```diff
@@ -111,8 +111,11 @@ def update(self, can_parsers) -> structs.CarState:
     ret.stockLkas = cp_ap_party.vl["DAS_steeringControl"]["DAS_steeringControlType"] == 2  # LANE_KEEP_ASSIST
 
     # Stock Autosteer should be off (includes FSD)
-    ret.invalidLkasSetting = cp_ap_party.vl["DAS_settings"]["DAS_autosteerEnabled"] != 0
-
+    if self.CP.carFingerprint in (CAR.TESLA_MODEL_3, CAR.TESLA_MODEL_Y):
+      ret.invalidLkasSetting = cp_ap_party.vl["DAS_settings"]["DAS_autosteerEnabled"] != 0
+    else:
+      #TODO: Find S/X equivalent
+      pass
```

Comment:
@ssysm let's set the `dashcamOnly` flag True for Model X and leave a comment explaining it's pending this. once that's done, I think we can merge this